doc/community/developer-guide.md

```diff
@@ -135,3 +153,47 @@ with pytest.raises(ZeroDivisionError) as excinfo:
 ```{code-cell} ipython3
 assert str(excinfo.value) == "division by zero"
 ```
+
+## Integration tests
+
+Integration tests check compatibility with different databases. They are executed on
+each PR; however, you might need to run them locally.
+
+```{note}
+Setting up the development environment for running integration tests locally
+is challenging given the number of dependencies. If you have problems,
+[message us on Slack.](https://github.com/orgs/community/discussions/26940)
```

Comment:
This is not the link to our Slack channel

Reply:
> Added 2 notes, besides that lgtm.
> 
> Some notes on the execution itself:
> 
> After executing `pytest src/test/integration` on Windows 11, the execution got stuck at this point:
> 
> ![image](https://user-images.githubusercontent.com/103026689/230320831-910e02ff-0ccf-4317-8962-0f5aa6422834.png)
> 
> Can't stop it/abort it (need to restart the command prompt)
> 
> This is the docker status
> 
> ![image](https://user-images.githubusercontent.com/103026689/230321351-6b2c4019-d938-4fc4-a75a-86c656d21f21.png)

opened: https://github.com/ploomber/jupysql/issues/382

Reply:
fixed link